libs/src/sdk/types/HashId.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod'
+import { decodeHashId } from '../utils/hashId'
+
+export const HashId = z.string().transform<number>((data: string) => {
```

Comment:
kind of a nit, but wonder if we should move this to a separate dir (since not really a type)

Reply:
I was thinking about this - wonder if we should keep schemas / types separate or lump them together like this?

Reply:
I kinda want to keep them colocated because the schema and type are so tightly coupled (type is derived from the schema)